Client/src/Components/NavBar/index.css

```diff
@@ -30,3 +30,9 @@
   height: 16px;
   color: var(--env-var-color-25);
 }
+
+#bw-uptime-logo-dashboard {
+  height: 16px;
+  float: left;
+  margin-left: 15px;
```

Comment:
The float is not needed here.

Reply:
Yes, this doesn't seem to actually do anything?

Reply:
Yep. It was just a test. Forgot to remove!